Develappers.BillomatNet/InvoiceService.cs

```diff
@@ -605,5 +607,67 @@ public async Task<InvoicePayment> CreatePaymentAsync(InvoicePayment value, Cance
                 throw new ArgumentException("wrong input parameter", nameof(value), wex);
             }
         }
+
+
```

Comment:
Develappers.BillomatNet/InvoiceService.cs#L611
The code must not contain multiple blank lines in a row. (SA1507)